test-harness/infratests/unit.go

```diff
@@ -136,7 +136,7 @@ func validatePlanResourceKeyValues(fixture *UnitTestFixture, plan *terraformCore
 			}
 		}
 	}
-
+	fmt.Println(seen)
```

Comment:
We have a task to add this debugging (https://github.com/microsoft/orion/issues/21). I think we should focus the effort for that task as proper logging will be more than dumping the output here. My suggestion is to remove this line.

Reply:
removed